scripts/project.py

```diff
@@ -17,11 +17,13 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
+
+import sys
 import abc
 import json
 import logging
-import pathlib
 import subprocess
+import pathlib
 import tempfile
```

Comment:
Imports were sorted, try to keep it this way.

Reply:
Ok.